package.json

```diff
@@ -33,6 +33,7 @@
     "jest-docblock": "21.3.0-beta.1",
     "jest-validate": "21.1.0",
     "leven": "2.1.0",
+    "markdown-magic": "^0.1.19",
```

Comment:
pin?

Reply:
Gah, I keep doing this. Need to figure out why my yarn isn't respecting `.yarnrc`.